src/uct/ib/ud/base/ud_ep.h

```diff
@@ -248,6 +248,7 @@ struct uct_ud_ep {
     uint32_t         conn_id;      /* connection id. assigned in connect_to_iface() */
     uint16_t         flags;
     uint8_t          path_bits;
+    uint8_t          rx_creq_count; /* TODO: remove when reason for DUP/OOO CREQ is found */
```

Comment:
object size tests may fail

Reply:
It should not, I added this to the hole. EP size remains to be 240